Sources/Mixpanel.m

```diff
@@ -502,7 +531,11 @@ - (void)track:(NSString *)event properties:(NSDictionary *)properties
     if (![MixpanelPersistence loadAutomaticEventsEnabledFlagWithApiToken:self.apiToken] && [event hasPrefix:@"$ae_"]) {
         return;
     }
-
+    
+#if defined(DEBUG)
+    if (![event hasPrefix:@"$"]) [[NSUserDefaults standardUserDefaults] setBool:YES forKey:@"MPDebugTrackedKey"];
```

Comment:
nit: prefer
```
if () {

} 
```

Reply:
actually `NSInteger += BOOL` works as expected, `YES=1` and `NO=0`